source/_docs/cms-api.md

```diff
@@ -0,0 +1,173 @@
+</div>
+
+## WordPress REST API
+
+Since Wordpress 4.7, the [REST API plugin](https://wordpress.org/plugins/rest-api/) is no longer needed as it is now part of Core.
+The complete list of resources about WP REST API is found in the [handbook](https://developer.wordpress.org/rest-api/)
+
+* To extract site posts for example, send a `GET` request to `/wp-json/wp/v2/posts`.
+
+* To update a user with ID 4: Send a `PUT` request to `/wp-json/wp/v2/users/4`.
+
+* To extract site posts but filter with a search term “awesome”: `GET /wp-json/wp/v2/posts?filter[s]=awesome`
+
+* To get involved or see older versions of the plugin, visit the [WP REST API Team site](http://v2.wp-api.org/).
+
+## Other Frameworks, Builds or Distributions
+
+You can [use custom upstreams](/docs/custom-upstream/), [make your own build](/docs/guides/build-tools/) or [install distributions](/docs/start-state/) that may serve as a CMS API.
+
+Unless there are some platform related issues, Pantheon does not support features or the code behind custom applications you may use.
+
+### Contenta CMS
+
+[Contenta](http://www.contentacms.org/) is an API-First Drupal distribution. It provides a standard platform that is API ready for building front-end applications. Contenta intends to ease the pain of using, or simply trying, decoupled Drupal.
+
+To get started, see the distro [Github repository](https://github.com/contentacms/contenta_jsonapi) and learn how to install.
+The demo site is hosted in Pantheon via the link: [http://live-contentacms.pantheonsite.io](http://live-contentacms.pantheonsite.io)
+
+You may see the sample repository of [Contenta CMS Demo here](https://github.com/contentacms/contenta_jsonapi_demo). It uses the [Composer Drops 8](https://github.com/pantheon-systems/example-drops-8-composer) example repository as base.
+
+
+## Deploying your CMS API site in Pantheon
+
+There are no differences when deploying a CMS API than a normal site. You may follow our [Launch Essentials](/docs/guides/launch/) guide to get started, and also refer to the [Pantheon Workflow](/docs/pantheon-workflow) page for how to deploy your code or manage your content.
+
+Make sure to check the [Best Practice Checklist for Going Live](/docs/guides/launch/next-steps/) and activate other features like [New Relic](/docs/new-relic/) for monitoring, or enabling [Solr](/docs/solr) and [Redis](/docs/redis/)
+
+You can also write scripts in [Guzzle](http://docs.guzzlephp.org/en/latest/), which is a great tool (and included in Drupal 8 Core) for testing. You can also use cURL via the command line or PHP.
+
+### Pantheon Global CDN
+
+All sites will benefit from the use of [Global CDN](/global-cdn/), a core platform offering for improved performance and security. It enables an advanced caching mechanism, and have free and automated HTTPS by default.
+
+To learn more, read our [Global CDN documentation](/docs/global-cdn/).
+
+<!-- How does Global CDN cache REST API resources -->
```

Comment:
What's the status of this commented section?

Reply:
I don't have complete info yet, but would like to get help from other folks for one of these sections